task/backend/task.go

```diff
@@ -0,0 +1,31 @@
+package backend
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb"
+)
+
+// TaskControlService is a low-level controller interface, intended to be passed to
+// task executors and schedulers, which allows creation, completion, and status updates of runs.
+type TaskControlService interface {
+	// CreateNextRun attempts to create a new run.
+	// The new run's now is assigned the earliest possible time according to task's cron,
```

Comment:
Reading through this again, I see that we are inconsistently using the word `now`.

`influxdb.Run` has a `ScheduledFor` field. `task/backend.QueuedRun` calls it `Now`.

I think in general `ScheduledFor` is more descriptive than `Now`.

But the `now` argument to `CreateNextRun` really is "now", the current time that the scheduler thinks it is. That `now` can stay, or if you have a better name for it, I'm open to that too.